app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+angular.module("openshiftConsole")
+  /**
+   * Widget for entering PVC information
+   *
+   * model:
+   *   The model for the input. The model will either use or add the following keys:
+   *     {
+   *       name: "",
+   *       accessMode: "",
+   *       Capacity: ""
+   *     }
+   *
+   */
+  .directive("oscPersistentVolumeClaim", function(){
+    return {
+      require: '^form',
+      restrict: 'E',
+      scope: {
+        route: "=model",
+        services: "=",
+        showNameInput: "=",
+        routingDisabled: "="
```

Comment:
You can probably don't need any of these three scope properties (services, showNameInput, or routingDisabled). The fieldset in the create form should take care of disabling the inputs, and we should always show the name input.